src/main/java/com/nike/cerberus/domain/IamPrincipalPermission.java

```diff
@@ -36,7 +36,7 @@
     @NotBlank(message = "IAM_ROLE_ROLE_ID_INVALID", groups = {Default.class, Updatable.class})
     private String roleId;
 
-    @Pattern(regexp = AWS_IAM_PRINCIPAL_ARN_REGEX, message = "SDB_IAM_PRINCIPAL_PERMISSION_ARN_INVALID", groups = {Default.class, Updatable.class})
+    @Pattern(regexp = AWS_IAM_PRINCIPAL_ARN_REGEX_ALLOWED, message = "SDB_IAM_PRINCIPAL_PERMISSION_ARN_INVALID", groups = {Default.class, Updatable.class})
```

Comment:
Restrictive regex prevents additional instance-profile ARNs - we don't want them used any more because we couldn't use them in KMS key policies so the only worked when we could convert to a role ARN